### ISSUE TITLE
Feature request everything else already has.

### ISSUE THREAD

**Issue by DarkAlchy** [kohya-ss/sd-scripts#650]
Simple.  Start saves beginning at XX.  Save last XX epochs.  Those two things are severely needed with your scripts.  Right now I miss it from Shiv's, and Thelastben's Dreambooth as I have a lot of epochs that I narrowed it down from X to Y but having to save every epoch is crazy when all I want are the saves between those two epochs.

Thank you.

**Comment by DamonianoStudios** [comment]
I agree that this would be a very useful feature, Derrian's Easy_LoRA_Training_Scripts does the second request and it's very useful.

**Comment by bmaltais** [comment]
Save last XX epochs already exist... `--save_last_n_steps="2"`

**Comment by bmaltais** [comment]
Hummm... just tried the option and it did not delete the extra models

**Comment by DarkAlchy** [comment]
Yep, and how about (addressing Kohya with this one) the first one that has been in every dreambooth since 1.4 at least where it doesn't save at all it begins saving where I tell it to start saving.  Seems really a bit backwards to save the last XX only because that makes deleting just an extra step to wear out the electronic drives even faster.  I never under stood that save last XX illogic.

BTW, "Save last XX epochs already exist... --save_last_n_steps="2""  why is the flag in steps while the saves are epochs?
